.travis.yml

```diff
@@ -12,12 +12,12 @@ julia:
 matrix:
   allow_failures:
     - julia: nightly
-env:
-  - GROUP=Core
-  - GROUP=Min
-  - GROUP=EquibrilateI
-  - GROUP=EquibrilateII
-  - GROUP=Misc
+#env:
```

Comment:
remember to put these back in place?

Reply:
Do you mean the equilibrate tests or the groups? The former or not valid anymore since that doesn't work with the new MTK implementation. Or do we want to use groups again? The current time is about 40 minutes, which I guess is borderline.

Reply:
oh, if the total time is now 40 minutes, then yeah we're good.